SSACloudShellMethod1.ps1

```diff
@@ -11,39 +11,56 @@ $selected_subscription_id = Read-Host -Prompt "`n Enter the subscription_id you
 Write-Host "Selected Subscription ID is: $selected_subscription_id"
 Set-AzContext -Subscription $selected_subscription_id
 
+$PolicyName = "DefenderForServerless"
+$PolicyScope = "/subscriptions/$($selected_subscription_id)"
+
 # For each function from $function_app_list, 
 # set AZURE_FUNCTIONS_SECURITY_AGENT_ENABLED to 1 to enable the agent.
 # remove AZURE_FUNCTIONS_SECURITY_AGENT_ENABLED to disable the agent.
-$toggle_option =  Read-Host -Prompt "`n Enter 0 to Disable, 1 to Enable, any other input will do nothing and exit.`n Note: enablement/disablement of the defender will result in function restart...."
-
-if($toggle_option -eq 1){
```

Comment:
Seems like this got removed unintentionally, can we undo this?

Reply:
Done